Tests/ExamplesTest/expected_output/httpanalyzer_manyprotocols.txt

```diff
@@ -45,8 +45,8 @@ HTTP request methods
 ---------------------
 | Method    | Count |
 ---------------------
-| GET       | 150   |
```

Comment:
@seladb Do you think this should be left untouched and only have more code added to fix the sorting the output?